ext/ddtrace.c

```diff
@@ -2111,7 +2237,84 @@ PHP_FUNCTION(DDTrace_set_user) {
         ZEND_HASH_FOREACH_END();
     }
 
-    RETURN_NULL();
+    if (user_id != NULL) {
+        _ddtrace_set_user(user_id, metadata, false);
+    }
+}
+
+PHP_FUNCTION(datadog_appsec_v2_track_user_login_failure) {
+    UNUSED(execute_data);
+
+    zend_string *login = NULL;
+    zend_bool exists;
```

Comment:
```suggestion
    zend_bool exists = false;
```

Not sure if it'll be initialised otherwise.